### DISCUSSION TITLE
The wundergroundpws does not support platform setup

### DISCUSSION THREAD

**Discussion by felloadmin** [cytech/Home-Assistant-wundergroundpws#166] [Q&A]
Hi,
i updated to v.2.0.1 an now i received this error message:
 
T**he wundergroundpws platform for the sensor integration does not support platform setup. Please remove it from your config.**

i have absolutly no idea how to handle it, any ideas on this?

sorry if this is the wrong way to ask.

Best regads and thanks

**Comment by cytech** [comment]
https://github.com/cytech/Home-Assistant-wundergroundpws/blob/v2.X.X/README.md

**Comment by cytech** [comment]
closed as outdated
